src/api/submission/validations2/benthic_photo_quadrat_transect.py

```diff
@@ -286,28 +316,30 @@
         validation_level=RECORD_LEVEL,
         validation_type=VALUE_VALIDATION_TYPE,
     ),
+]
+
+bpqt_classification_validations = bpqt_base_validations + [
     Validation(
-        validator=UniqueQuadratTransectValidator(
-            protocol_path="data.protocol",
-            label_path="data.quadrat_transect.label",
-            number_path="data.quadrat_transect.number",
-            relative_depth_path="data.quadrat_transect.relative_depth",
-            site_path="data.sample_event.site",
-            management_path="data.sample_event.management",
-            sample_date_path="data.sample_event.sample_date",
-            depth_path="data.quadrat_transect.depth",
-            observers_path="data.observers",
+        validator=ImageCountValidator(
+            num_quadrats_path="data.quadrat_transect.num_quadrats",
         ),
-        paths=[
-            "data.quadrat_transect.label",
-            "data.quadrat_transect.number",
-            "data.quadrat_transect.relative_depth",
-            "data.quadrat_transect.depth",
-            "data.sample_event.site",
-            "data.sample_event.management",
-            "data.sample_event.sample_date",
-        ],
+        paths=[],
         validation_level=RECORD_LEVEL,
         validation_type=VALUE_VALIDATION_TYPE,
     ),
+    Validation(
+        validator=CollectRecordImagesValidator(),
+        paths=["data.obs_benthic_photo_quadrats"],
```

Comment:
Might be good to sync up on this -- I thought all the validations for the ML case would not reference `obs_benthic_photo_quadrats` because they don't exist?